sdk/download.go

```diff
@@ -318,3 +319,18 @@ func UpdateSignedFile(file, url string, client *http.Client) error {
 	plog.Infof("Verified file: %s", file)
 	return nil
 }
+
+func DetectSDKVersion(url, branch string) (string, error) {
```

Comment:
Still need to add the `GetSDKVersion` to avoid the git clone if the repo tree is already there. Most commonly this would be the case if `create --replace` is used.